src/anndata/_core/index.py

```diff
@@ -26,14 +26,13 @@ def _normalize_indices(
     if isinstance(index, pd.Series):
         index: Index = index.values
     if isinstance(index, tuple):
-        if len(index) > 2:
-            raise ValueError("AnnData can only be sliced in rows and columns.")
-        # deal with pd.Series
-        # TODO: The series should probably be aligned first
-        if isinstance(index[1], pd.Series):
-            index = index[0], index[1].values
-        if isinstance(index[0], pd.Series):
-            index = index[0].values, index[1]
+        if len(index) == 2:
+            # deal with pd.Series
+            # TODO: The series should probably be aligned first
+            if isinstance(index[1], pd.Series):
+                index = index[0], index[1].values
+            if isinstance(index[0], pd.Series):
+                index = index[0].values, index[1]
```

Comment:
as said, why not just this?

```suggestion
        # TODO: The series should probably be aligned first
        index = tuple(i.values if isinstance(i, pd.Series) else i for i in index)
```